cpu_tests/test_streaming_iterators.py

```diff
@@ -40,11 +41,41 @@ def get_simple_dataset():
             torch.LongTensor([8, 9]),
         ]
     )
-    dataset = DeferredDataset(dataset)
-    dataset = SkipDeferredDataset(dataset, 0)
+    dataset = DocumentToSequenceDataset(
+        dataset,
+        block_size=None,
+        permute_documents=False,
+        break_mode="passthrough",
+        padding_idx=1,
+    )
     return dataset
 
 
+class FakeTensorData(torch.utils.data.Dataset):
+    def __init__(self):
+        self.rng = random.Random(0)
+        self.trng = torch.Generator()
+        self.trng.manual_seed(0)
+        self.items = [
+            torch.randint(
+                256, size=(self.rng.randrange(512, 513),), generator=self.trng
```

Comment:
Nit: is `randrange(512, 513)` still needed here? Or meant to be kept at `randrange(512, 8192)` as before?